fpsd/attack.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import datetime
+from itertools import product
+import pdb
+import pickle
+import yaml
+
+import classify, database
+
+
+def run(options):
+    """Takes an attack file, gets the features, and runs all experiments
+    and saves the output in the database.
+
+    Args:
+        options [dict]: attack setup file
+    """
```

Comment:
I was using [Google style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google), but don’t have strong feelings one way or the other so the docstring format you like there is fine w/me and from now on I will follow this format. However, most of the existing docstrings in this branch I wrote before that issue was filed so would rather not rewrite the Google style docstrings at this stage in the interests of time.